peribolos/knative-sandbox.yaml

```diff
@@ -707,11 +707,11 @@ orgs:
           Technical Oversight Committee:
             description: Members of the Knative Technical Oversight Committee (TOC)
             maintainers:
-            - evankanderson
```

Comment:
I think you also need to move Evan to the members list (and remove him from the admins one) in both knative and sandbox